### PR TITLE
Readme updates based on a fresher Ubuntu installation experience

### PR DESCRIPTION
**Pull request by yeputons**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +117,14 @@ cd test_runner
 pytest
 ```
 
+If you see a `Found interfering processes running` error, stop pageserver and postgres nodes for all branches:
```
Comment (SomeoneToIgnore) [inline README.md:120]: This error message is from the test runner that has its own readme, we can put it there instead: https://github.com/zenithdb/zenith/blob/main/test_runner/README.md

I've been suggested once to do the `killall -9 postmaster pageserver wal_acceptor postgres` to fix the error, we can add the command too.

**Review by SomeoneToIgnore — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (yeputons) [reply README.md:120]: I think it's better to put it directly in the error message, PTAL. I would be wary of `killall`, maybe someone will try to run tests on a non-isolated production server.

The script does not check for `postmaster`, does not?

File: README.md [hunk 2]
```diff
@@ -117,6 +117,14 @@ cd test_runner
...
+
+```sh
+./target/debug/zenith pg stop migration_check
+./target/debug/zenith pg stop main
+./target/debug/zenith stop
```
Comment (kelvich) [inline README.md:125]: That wouldn't help with system-wide postgres. I'd rather explain what processes we are talking about in error message. Something like `Found interfering processes running (that is 'postgres' or 'wal_acceptor' or 'pageserver')`.

I hope we may ditch that check at all after https://github.com/zenithdb/zenith/pull/353 will be merged

Comment (kelvich) [reply README.md:125]: Also `migration_check` is test specific branch name.

File: README.md (hunk 1 above)
Comment (SomeoneToIgnore) [reply README.md:120]: Nice idea about the error message.

The `killall` one does not check for anything, no.

File: README.md (hunk 2 above)
Comment (yeputons) [reply README.md]: That's true. I believe the error message is already clear enough.

Here I wanted to streamline the "opened README and want to check that everything is running without going into details" situation. Here is another take, PTAL.

**Review by kelvich — APPROVED** [review]

**Pull request merged**